Author: Andreas Beckmann <anbe@debian.org>
Description: rename the nvidia kernel module for concurrent legacy installation

--- a/Kbuild
+++ b/Kbuild
@@ -35,7 +35,7 @@
 # ahead and append ".o" or ".ko" to form the final module name.
 #
 
-MODULE_NAME := nvidia
+MODULE_NAME := #MODULE#
 VERSION_HEADER := nv_compiler.h
 
 #
